.github/workflows/docs_light.yml

```diff
@@ -52,8 +52,8 @@ jobs:
         uses: actions/cache@v2
         with:
           path: .docker
-          key: ubuntu-docs-${{ hashFiles('cpp/**') }}
-          restore-keys: ubuntu-docs-
+          key: conda-docs-${{ hashFiles('docs/**') }}
+          restore-keys: conda-docs-
```

Comment:
Hmm, this change might partly be a mistake. Perhaps we want to compute the hashes on the C++ files?